src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java

```diff
@@ -203,6 +204,14 @@ public double getDefaultAlleleFrequency() {
     @Argument(fullName = MUTECT3_DATASET_LONG_NAME, optional = true, doc="Destination for Mutect3 data collection")
     public File mutect3Dataset;
 
+    /**
+     * VCF of known calls for a sample used for generating a Mutect3 training dataset.  Unfiltered variants contained
```

Comment:
Unfiltered here meaning PASS or `.`?
```suggestion
     * VCF of known calls for a sample used for generating a Mutect3 training dataset.  Passing variants contained
```

Reply:
done

Reply:
It means either.